src/MagentoHackathon/Composer/Magento/Plugin.php

```diff
@@ -79,6 +80,16 @@ class Plugin implements PluginInterface, EventSubscriberInterface
     protected $entryFactory;
 
     /**
+     * @var EventManager
+     */
+    protected $eventManager;
```

Comment:
No reason to make this protected. Hide as much as you can. Make it private. You have a public method exposing this anyway. And it's unlikely that this class will be extended.